psalm.xml

```diff
@@ -13,6 +13,7 @@
             <directory name="src/Fixture"/>
             <directory name="tests/Application"/>
             <directory name="vendor"/>
+            <file name="src/DependencyInjection/Configuration.php"/>
```

Comment:
You should not ignore this. Just add the applicable suppressions inside the file

Reply:
It raises a lot of new errors that were not present before. I suspect some update from Psalm.

Errors like this: 

```
ERROR: PossiblyUndefinedMethod - src/DependencyInjection/Configuration.php:47:27 - Method Symfony\Component\Config\Definition\Builder\NodeDefinition::arrayNode does not exist (see https://psalm.dev/108)
                        ->arrayNode('available_orientations')
```

And there are another 130 errors like this one... So I'm not sure about how to proceed

Reply:
Like this: https://github.com/Setono/SyliusImagePlugin/blob/master/src/DependencyInjection/Configuration.php#L27